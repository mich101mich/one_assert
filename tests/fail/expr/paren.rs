fn main() {
    one_assert::assert!((1));
}
