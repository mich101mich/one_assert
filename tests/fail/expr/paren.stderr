error[E0308]: mismatched types
 --> tests/fail/expr/paren.rs:2:26
  |
2 |     one_assert::assert!((1));
  |                          ^ expected `bool`, found integer
